test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/ExportDataValidationTests.cs

```diff
@@ -58,12 +61,19 @@ public async Task GivenFhirServer_WhenDataIsExported_ThenExportedDataIsSameAsDat
         private bool ValidateDataFromBothSources(Dictionary<string, string> dataFromServer, Dictionary<string, string> dataFromStorageAccount)
         {
             bool result = true;
+
             if (dataFromStorageAccount.Count != dataFromServer.Count)
             {
                 _outputHelper.WriteLine($"Count differs. Exported data count: {dataFromStorageAccount.Count} Fhir Server Count: {dataFromServer.Count}");
                 return false;
             }
 
+            if (dataFromStorageAccount.Count == 0)
```

Comment:
I don't think this check is necessary. If someone runs this test (as part of running all E2E tests, say, before sending out a PR) on their localbox with an empty fhir-server, then the test will fail and might confuse them. If `dataFromStorageAccount.Count == 0` and `dataFromServer.Count == 0` the test should still pass.

Reply:
I commented this check out, but I'm going to leave it in as I found it useful for me while editing the tests to make sure I didn't mess something up and no data was exported.